src/nat/front_ends/fastapi/fastapi_front_end_plugin_worker.py

```
@@ -1088,6 +1090,183 @@ async def redirect_uri(request: Request):
+
+                        mcp_clients_info.append({
+                            "function_group": group_name,
+                            "server": client.server_name,
+                            "transport": config.server.transport,
+                            "session_healthy": session_healthy,
+                            "tools": tools_info,
+                            "total_tools": len(configured_short_names),
+                            "available_tools": available_count
+                        })
+
+                    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)